packages/astropy/meta.yaml

```diff
@@ -0,0 +1,15 @@
+package:
+  name: astropy
+  version: 3.1.2
+
+source:
+  url: https://pypi.io/packages/source/a/astropy/astropy-3.1.2.tar.gz
```

Comment:
Would it be worth updating to astropy 4.0 ?

Reply:
Good point :) I just pushed last night to see if CI would run faster than my laptop build (it did)